docs/api/cases/cases-api-find-cases.asciidoc

```diff
@@ -0,0 +1,189 @@
+[[cases-api-find-cases]]
+== Find cases API
+++++
+<titleabbrev>Find cases</titleabbrev>
+++++
+
+Retrieves a paginated subset of cases. By default, the first page is returned
+with 20 results per page.
+
+=== Request
+
+`GET <kibana host>:<port>/s/<space_id>/api/cases/_find`
```

Comment:
I think we should put both endpoints one with the space and one without. As it is, a user may use `GET <kibana host>:<port>/s/api/cases/_find` (no `<space_id>` on the path because we say it is optional) and  get 404 error.

We can follow this example: https://www.elastic.co/guide/en/kibana/master/saved-objects-api-get.html

Reply:
Good idea!  Done in ed7b3331c93678e255350b47d6e0e31398421e23